tests/interface/test_dialogs.py

```diff
@@ -178,11 +178,7 @@ def microbit():
     return device
 
 
-@mock.patch(
-    "mu.interface.dialogs.ESPFirmwareFlasherWidget.esptool_is_installed",
-    return_value=True,
-)
-def test_ESPFirmwareFlasherWidget_setup(esptool_is_installed, microbit):
+def test_ESPFirmwareFlasherWidget_setup(microbit):
```

Comment:
Nice to see mocks are gone.

Reply:
Yeah, `esptool_is_installed()` wasn't really used anywhere, might have been accidentally left after a refactor at some point.